src/libraries/System.Drawing.Common/src/System/Drawing/DrawingComWrappers.cs

```diff
@@ -0,0 +1,331 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Drawing
+{
+    /// <summary>
+    /// The ComWrappers implementation for System.Drawing.Common's COM interop usages.
+    ///
+    /// Supports IStream and IPicture COM interfaces.
+    /// </summary>
+    internal unsafe class DrawingComWrappers : ComWrappers
+    {
+        private const int OK = 0;
```

Comment:
Perhaps consume the already existing constant:

https://github.com/dotnet/runtime/blob/5306472296c5c174be9379dac3920880de64c4a0/src/libraries/Common/src/Interop/Windows/Interop.HRESULT.cs#L7-L20

Reply:
There's a debate in my mind about using the `HRESULT` enum or `int`. Either way, it leads to casting, and I was trying to minimize the casting.

```C#
private static ??? Foo(IntPtr thisPtr)
{
    try
    {
        ...
    }
    catch (Exception e)
    {
        return e.HResult; // this is an `int`
    }

    return ??;
}
```

I chose to use `int` on the signatures solely because `return e.HResult` is in every method, and if I returned the `HResult` enum, it would mean I would need to cast on that line in every method.

So I made my own `int OK` const here, so I could return it at the end of every method.

I'll change this const declaration over to use the existing constant. But I wanted to write out my thinking here of why I chose `int` over `HRESULT` in all the signatures.